tests/test_wgpu_native_query_set.py

```diff
@@ -1,11 +1,15 @@
 import wgpu.utils
+import gc
 
 from testutils import run_tests, can_use_wgpu_lib
+from tests_mem.testutils import is_pypy
 from pytest import mark
 
 
 @mark.skipif(not can_use_wgpu_lib, reason="Needs wgpu lib")
 def test_query_set():
+    if is_pypy:
```

Comment:
this gets the test to pass - but doesn't seem to be the solution.

I tried to find out which specific test before this one causes the failure - but wasn't able to find any specific pair. It seems to just be volume of tests run before. As a subset like `-k native` would only crash on the 2nd run.

Reply:
LOL, if I run `pytest tests` on pypy, I get an abort in this test. If I remove this line, it passes all tests 🤡